eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -390,6 +394,38 @@ unRegisterDRep drep = do
       & bodyTxL . certsTxBodyL
         .~ SSeq.singleton (UnRegDRepTxCert drep refund)
 
+arbitraryUnRegTxCert ::
```

Comment:
I think that the `gen` prefix would be more consistent with the functions we already have that generate arbitrary values:  `genRegTxCert`  and `genRegTxCert`. 
However,  many of the functions in this module generate data for `Imp` tests , so I think even 
`unRegTxCert` and `regTxCert` names would work . Just a suggestion, feel free to ignore it.